src/components/form/form.scss

```diff
@@ -1,5 +1,9 @@
 @import "./../../style-config/colors";
 
+.carbon-form__left-aligned-actions {
```

Comment:
Nah, I only need this one to float the left aligned ones to the left. The right ones already float to the right.